app/assets/stylesheets/views/_brexit_checker_account_signup.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-experiment > .govuk-inset-text {
```

Comment:
TIL: If i'm not specific with the rule... all the insert text's that we use on the results page also go blue :smile: